pool/cqrs_command/session_command.ml

```diff
@@ -353,19 +353,41 @@ end = struct
 end
 
 module Delete : sig
-  include Common.CommandSig with type t = Session.t
+  include Common.CommandSig
+
+  type t =
+    { session : Session.t
+    ; follow_ups : Session.t list
+    ; templates : Message_template.t list
+    }
+
+  val handle
+    :  ?tags:Logs.Tag.set
+    -> t
+    -> (Pool_event.t list, Pool_common.Message.error) result
 
   val effects : Pool_common.Id.t -> Guard.Authorizer.effect list
 end = struct
-  type t = Session.t
+  type t =
+    { session : Session.t
+    ; follow_ups : Session.t list
+    ; templates : Message_template.t list
+    }
 
-  let handle ?(tags = Logs.Tag.empty) session =
+  let handle ?(tags = Logs.Tag.empty) { session; follow_ups; templates } =
+    let open CCFun in
     let open CCResult in
     Logs.info ~src (fun m -> m "Handle command Delete" ~tags);
-    (* TODO [aerben] how to deal with follow-ups? currently they just
-       disappear *)
-    let* () = Session.is_deletable session in
-    Ok [ Session.Deleted session |> Pool_event.session ]
+    if CCList.is_empty follow_ups |> not
```

Comment:
Please include this check in the `is_deletable` function to make sure we always use the same checks. 

See: https://github.com/uzh/pool/blob/main/pool/web/view/page/page_admin_session.ml#L278